packages/connect/src/base/listener.ts

```diff
@@ -764,56 +688,6 @@ class Listener extends EventEmitter implements InterfaceListener {
     // @TODO figure what to do if there is more than one address
     return usableInterfaces[0].address
   }
-
-  private async connectToRelay(relay: Multiaddr, timeout: number): Promise<number> {
```

Comment:
function is now split up in two parts, get a connection to some party (in `index.ts`) and do sth. application-specific (in e.g. `entry.ts` and `relay.ts`)